uint8 robot_id
geometry_msgs/Pose2D[] node_list
sensor_msgs/LaserScan[] laser_scans
std_msgs/Time[] timestamps
